cloudfoundry-client-lib/pom.xml

```diff
@@ -21,8 +21,8 @@
         <tomcat.version>8.0.52</tomcat.version>
         <jetty.version>8.1.12.v20130726</jetty.version>
         <byteman.version>2.1.3</byteman.version>
-        <cloudfoundry.v3.client.version>3.15.0.RELEASE</cloudfoundry.v3.client.version>
         <immutables.version>2.7.5</immutables.version>
+        <cloudfoundry.v3.client.version>3.16.1.BUILD-SNAPSHOT</cloudfoundry.v3.client.version>
```

Comment:
Ignore this. There is no client version available with these changes still. It will be changed when the capi accepts my pull request.